lib/netutil/tls.go

```diff
@@ -22,10 +29,18 @@ func GetServerTLSConfig(tlsCertFile, tlsKeyFile string, tlsCipherSuites []string
 	if err != nil {
 		return nil, fmt.Errorf("cannot use TLS cipher suites from tlsCipherSuites=%q: %w", tlsCipherSuites, err)
 	}
+	minVersion, err := tlsVersionFromName(minTLSVersion)
+	if err != nil {
+		return nil, fmt.Errorf("cannnot use TLS min version from minTLSVersion=%q: %w", minTLSVersion, err)
+	}
+	maxVersion, err := tlsVersionFromName(maxTLSVersion)
+	if err != nil {
+		return nil, fmt.Errorf("cannnot use TLS max version from maxTLSVersion=%q: %w", minTLSVersion, err)
```

Comment:
add check for minVersion > maxVersion. If so, maxVersion = minVersion.

Reply:
fixed